packages/react-components/react-image/src/components/Image/Image.test.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { isConformant } from '../../testing/isConformant';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28509&commitId=e64469f6874bbdc768a0f961d546e9bbc5b52496&env=prod&clientType=FLUENTUI)


  <details><summary><b>Field</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.RadioGroup.default.chromium.png|55125| Changed|
|Field.Select.default.chromium.png|26050| Changed|

</details>


  <details><summary><b>Radio Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio Converged.group-vertical.default.chromium.png|20409| Changed|
|Radio Converged.group-disabled - RTL.default.chromium.png|20400| Changed|
|Radio Converged.group-disabled.default.chromium.png|20399| Changed|
|Radio Converged.group-vertical - RTL.default.chromium.png|20410| Changed|

</details>


  <details><summary><b>Select Converged</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Select Converged.Appearance- outline (default).default.chromium.png|3698| Changed|
|Select Converged.Appearance- underline.default.chromium.png|3696| Changed|
|Select Converged.Custom Icon.default.chromium.png|3737| Changed|
|Select Converged.Disabled.default.chromium.png|3278| Changed|
|Select Converged.Invalid- filled-darker.focused.chromium.png|8071| Changed|
|Select Converged.Invalid- filled-lighter.default.chromium.png|8176| Changed|
|Select Converged.Invalid- filled-lighter.focused.chromium.png|8188| Changed|
|Select Converged.Invalid- filled-darker.hover.chromium.png|8060| Changed|
|Select Converged.Invalid- filled-lighter.hover.chromium.png|8176| Changed|
|Select Converged.Invalid- outline.default.chromium.png|3703| Changed|
|Select Converged.Invalid- outline.focused.chromium.png|4101| Changed|
|Select Converged.Invalid- outline.hover.chromium.png|3703| Changed|
|Select Converged.Invalid- underline.default.chromium.png|3690| Changed|
|Select Converged.Invalid- underline.hover.chromium.png|3690| Changed|
|Select Converged.Appearance- outline (default).focused.chromium.png|4101| Changed|
|Select Converged.Size- small.default.chromium.png|3668| Changed|
|Select Converged.Size- large.default.chromium.png|6643| Changed|
|Select Converged.Appearance- filled-lighter.focused.chromium.png|8188| Changed|
|Select Converged.With value.default.chromium.png|3760| Changed|
|Select Converged.Appearance- outline (default).hover.chromium.png|3699| Changed|
|Select Converged.Appearance- underline.hover.chromium.png|3696| Changed|
|Select Converged.Appearance- filled-darker.default.chromium.png|8071| Changed|
|Select Converged.Invalid- underline.focused.chromium.png|4099| Changed|
|Select Converged.Appearance- underline.focused.chromium.png|4099| Changed|
|Select Converged.Appearance- filled-lighter.hover.chromium.png|8188| Changed|
|Select Converged.Appearance- filled-lighter.default.chromium.png|8188| Changed|
|Select Converged.Appearance- filled-darker.focused.chromium.png|8071| Changed|
|Select Converged.With appearance override.default.chromium.png|7843| Changed|
|Select Converged.Appearance- filled-darker.hover.chromium.png|8071| Changed|
|Select Converged.Invalid- filled-darker.default.chromium.png|8060| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>